packages/ui/test/overview/components/ProposalsOverview.test.tsx

```diff
@@ -0,0 +1,172 @@
+      votingRound: 1,
+    },
+    {
+      id: '1',
+      voteKind: 'APPROVE',
+      network: 'OLYMPIA',
+      createdAt: '2021-12-04T07:57:36.653Z',
+      voterId: '1',
+      inBlock: 1592,
+      rationale:
+        '# nemo ea est in vitae\n\nAb debitis minima laboriosam velit dolorem omnis et. Ad dolorem inventore qui dolor minus voluptatem facere est modi. Mollitia modi vel sit. Officiis ut reiciendis accusamus. Maxime vel amet enim occaecati architecto maiores. Soluta quaerat saepe est consequuntur odit tempora consequatur ducimus.\n\n## amet architecto et\n\nEst asperiores odit. Et voluptates quae adipisci. Porro accusantium perspiciatis omnis ratione excepturi reprehenderit ipsa quia. Fugit aut et quam voluptate sed sit.\n \rDeleniti ut cum id hic ullam ipsum repudiandae molestiae distinctio. Aut tenetur pariatur voluptas eum aut labore incidunt. Id numquam blanditiis voluptatem ut soluta error consequatur. Labore facere porro tempora. Vitae non aspernatur accusantium voluptas quia pariatur nulla rerum. Ullam enim incidunt ducimus quia.\n \rAnimi iure magnam voluptatem culpa nihil veniam est. Excepturi quod unde et deserunt praesentium nulla repellat quas. Voluptas nam corrupti autem exercitationem. Aspernatur veritatis quia nostrum fugiat aut.',
+      votingRound: 1,
+    },
+  ],
+}
+
+const dormantProposalMock = {
+  ...baseMock,
+  id: '4',
+  title: 'Dormant Test Proposal',
+  status: 'dormant',
+  createdAt: '2021-07-21T10:00:00.000Z',
+  statusSetAtTime: '2021-07-24T10:00:00.000Z',
+  details: {
+    type: 'fundingRequest',
+    data: {
+      destinationsList: {
+        destinations: [
+          {
+            account: '5GETSBUMwbLJgUTWMQgU8B2CP7E8kDHR8NoNNZh5tqums9AF',
+            amount: 5000,
+          },
+        ],
+      },
+    },
+  },
+  councilApprovals: 1,
+}
+
+describe('UI: Proposals overview', () => {
+  const mockServer = setupMockServer({ noCleanupAfterEach: true })
+  const api = stubApi()
+
+  beforeEach(() => {
+    seedMembers(mockServer.server, 2)
+  })
+
+  describe('Displays proper number of', () => {
+    beforeEach(() => {
+      testProposals.map((proposal) => seedProposal(proposal, mockServer.server))
+      renderComponent()
+    })
+
+    it('New proposals', async () => {
+      expect((await screen.findByText('proposals.new')).previousSibling?.textContent).toBe('1')
+    })
+
+    it('Proposals approved', async () => {
+      expect((await screen.findByText('proposals.approved')).previousSibling?.textContent).toBe('3')
+    })
+
+    it('Proposals rejected', async () => {
+      expect((await screen.findByText('proposals.rejected')).previousSibling?.textContent).toBe('2')
+    })
+  })
+
+  describe('Proposal in Deciding stage', () => {
+    beforeEach(() => {
+      stubProposalConstants(api)
+      seedProposal(decidingProposalMock, mockServer.server)
+      renderComponent()
+    })
+
+    it('Displays title', async () => {
+      expect(await screen.findByText('Deciding Test Proposal'))
```

Comment:
If you have this proposal mocked on test file, it might be better - more typo resistance to use `decidingProposalMock.title` here as string